### PR TITLE
Julia RISCV

### PR DESCRIPTION
**Pull request by IgorErin**
- riscv asm gen

### CODE REVIEW

File: src/julia.cpp [hunk 1]
```diff
@@ -65,7 +65,7 @@ struct HComplex {
 void halide_julia(uint8_t* dst, int height, int width) {
     Buffer output(dst, {width, height});
 #ifdef __riscv
-   // add later
+    halide_julia_rv(output);
```
Comment (dkurt) [inline src/julia.cpp:68]: `halide_julia(output)`

**Comment by IgorErin** [comment]
*86 benchmark*

**Comment by dkurt** [comment]
Please apply this commit to your PR: https://github.com/dkurt/halide_riscv/pull/21/commits/0b049b934d123adb909b483678e4760bb1f59bb0

Including generated `halide_runtime.o` to `aot` folder.


Now it runs: https://github.com/dkurt/halide_riscv/actions/runs/7813223639/job/21312408178?pr=21
```
[==========] Running 11 tests from 6 test cases.
[----------] Global test environment set-up.
[----------] 2 tests from julia
[ RUN      ] julia.halide
[ PERFSTAT ]    (samples=10   mean=718.47   median=712.29   min=706.54   stddev=20.12 (2.8%))
[       OK ] julia.halide (7185 ms)
[ RUN      ] julia.reference
[ PERFSTAT ]    (samples=31   mean=4.27   median=4.21   min=4.15   stddev=0.13 (3.0%))
[       OK ] julia.reference (134 ms)
[----------] 2 tests from julia (7319 ms total)
```

**Comment by dkurt** [comment]
With `julia.compute_at(result, y)`:
```
[----------] 2 tests from julia
[ RUN      ] julia.halide
[ PERFSTAT ]    (samples=10   mean=369.13   median=366.40   min=355.97   stddev=10.11 (2.7%))
[       OK ] julia.halide (3692 ms)
[ RUN      ] julia.reference
[ PERFSTAT ]    (samples=14   mean=4.27   median=4.25   min=4.17   stddev=0.13 (3.0%))
[       OK ] julia.reference (61 ms)
[----------] 2 tests from julia (3753 ms total)
```

**Pull request merged**